static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx

```diff
@@ -46,6 +46,7 @@ type ItunesRevalidationSessionContext = SessionContext & {
 };
 
 type InitialData = {
+  type: string;
```

Comment:
moved this up here so it's easier to just compare this side-by-side against `CustomRepoAppStoreConnect`.